production/helm/loki/templates/deployment.yaml

```diff
@@ -63,6 +63,9 @@ spec:
             {{- toYaml .Values.resources | nindent 12 }}
           securityContext:
             readOnlyRootFilesystem: true
+          env:
+            - name: JAEGER_AGENT_HOST
```

Comment:
I mean to use the if not empty on `.Values.jaegerAgentHost` and move it inside env.

Reply:
Test it, with empty value do no harm.